config/locales/en.yml

```diff
@@ -13,9 +13,6 @@ en:
       sessions_controller:
         new:
           invalid_email_or_password: Incorrect email or password. After 5 login attempts, accounts are locked for 30 minutes.
-  email:
-    user_message:
-      subject: "Update from GetYourRefund"
```

Comment:
Why did this get deleted? We'd have thought that any unused keys would be caught by the i18n specs already